scripts/benchmark/checks/deploy_dos.py

```diff
@@ -0,0 +1,296 @@
+import logging
+import pathlib
+from typing import (
+    Tuple
+)
+
+from web3 import (
+    Web3
+)
+
+from eth_utils import (
+    encode_hex,
+    decode_hex,
+)
+
+from eth.constants import (
+    CREATE_CONTRACT_ADDRESS
+)
+from eth.chains.base import (
+    MiningChain,
+)
+from eth.rlp.blocks import (
+    BaseBlock,
+)
+
+from .base_benchmark import (
+    BaseBenchmark,
+)
+from utils.chain_plumbing import (
+    FUNDED_ADDRESS,
+    FUNDED_ADDRESS_PRIVATE_KEY,
+    get_all_chains,
+)
+from utils.compile import (
+    get_compiled_contract
+)
+from utils.reporting import (
+    DefaultStat,
+)
+from utils.tx import (
+    new_transaction,
+)
+
+from abc import (
```

Comment:
I hate to be nitpicking about these things and in the future we'll probably use something like `black` to auto format but for the time being, can you move this import to the very top. Imports should be ordered according to our style guide: https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#overall-ordering